tripy/docs/conf.py

```diff
@@ -178,40 +187,40 @@ def process_docstring(app, what, name, obj, options, lines):
                     ":returns:" in doc
                 ), f"For: {obj}, return value is not documented. Please ensure you've included a `Returns:` section"
 
-    # new docstring logic:
-    # first figure out if we should it is the new docstring
+    # New docstring logic:
+    # First figure out if we should it is the new docstring.
     if name.split(".")[-1] in TYPE_VERIFICATION.keys():
         cleaned_name = name.split(".")[-1]
```

Comment:
Unqualified name might be more descriptive than cleaned name:
```suggestion
    unqual_name = name.split(".")[-1]
    if  unqual_name in TYPE_VERIFICATION.keys():
```